doc/performance-investigations/0012-increase-machine-type-core-count/summary.md

```diff
@@ -0,0 +1,49 @@
+# Investigate Machine Type Architecture Performance
+
+EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of machine type on the performance of the runner application, focusing on the number of CPU cores as Runner is primarily a CPU bound application with limited network IO to Datastore and Redis.
```

Comment:
I'm not sure about limited network IO, the calls to Datastore are up to 50% of the request time.

Reply:
How about `EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of machine type on the performance of the runner application, focusing on the number of CPU cores provisioned` ?

Reply:
After re-reading this, I think it should be clearer about what we're testing. Something like: 
> This investigation looked at the impact of machine type on the performance of the runner application, specifically the impact of using larger instances with more pods.